docs/source/getting_started/advanced.rst

```diff
@@ -46,6 +51,8 @@ show examples of when such invariances can be useful.
 
 - **Random Grayscale** E.g. We don't care about the color of a tree
 
+- **Solarization** E.g. We don't care about color and brightness
```

Comment:
Could you describe solarisation a bit more? E.g. with an example for when it would be useful?

Reply:
I don't know what else to add here. Solarization is just inverting colors above a threshold.